integration/src/test/java/org/kie/hacep/PodAsLeaderTest.java

```diff
@@ -181,7 +180,7 @@ public void processMessagesAsLeaderAndCreateSnapshotTest() {
             assertTrue(snapshot.getLastInsertedEventOffset() > 0);
             assertFalse(snapshot.getFhMapKeys().isEmpty());
             assertNotNull(snapshot.getLastInsertedEventkey());
-            assertTrue(snapshot.getFhMapKeys().size() == 8);//@TODO fix number of FH before the snapshot
+            assertTrue(snapshot.getFhMapKeys().size() == 9);
```

Comment:
Use `assertEquals(9, snapshot.getFhMapKeys().size())` so in case of failure we will see directly in the log, actual and expected value

Reply:
ok